AGENTS.md

```
@@ -0,0 +1,533 @@
+- Unit tests verify logic, E2E tests verify the action actually works
+- Tests run against your PR branch code (not main)
+- Catches issues like missing `dist/` rebuilds, runtime errors, config parsing bugs
+
+**Testing Approach:**
+
+This project uses **CI-first testing**:
+1. Local: `npm run all` (unit tests, linting, bundling)
+2. CI: Push to PR → automatic E2E test with `uses: ./`
+
+**Local Testing:**
+```bash
```

markdownlint-cli2 (0.20.0)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)